.github/actions/build-and-push-to-quay/action.yml

```diff
@@ -22,10 +22,18 @@ inputs:
     description: 'Should we tag with calender versioning?'
     required: false
     default: false
+  calver_suffix:
+    description: 'Optional suffix to the calendar version'
+    required: false
+    default: ""
```

Comment:
did not end up consuming this feature in a workflow, but might as well leave it in case we want this flag later.